trieste/models/gpflux/builders.py

```diff
@@ -154,3 +155,92 @@ def _get_data_stats(data: Dataset) -> tuple[TensorType, TensorType, int]:
     num_data_points = len(data.observations)
 
     return empirical_mean, empirical_variance, num_data_points
+
+
+def build_vanilla_flexible_deep_gp(
+    dim_output: int,
+    data: Dataset,
+    search_space: SearchSpace,
+    num_layers: int = NUM_LAYERS,
+    num_inducing_points: Optional[int] = None,
+    inner_layer_sqrt_factor: float = INNER_LAYER_SQRT_FACTOR,
+    likelihood_variance: float = LIKELIHOOD_VARIANCE,
+    trainable_likelihood: bool = True,
+) -> DeepGP:
+    """
+    Build a :class:`~gpflux.models.DeepGP` model with sensible initial parameters. We found the
+    default configuration used here to work well in most situation, but it should not be taken as a
+    universally good solution.
```

Comment:
whats the key difference with the ohter builder?

Reply:
commented a bit below